docs/release-notes/0.11/release-0.11.md

```diff
@@ -0,0 +1,87 @@
+# .NET for Apache Spark 0.11 Release Notes
+
+### New Features and Improvements
+
+* Ability to pass and return [corefxlab](https://github.com/dotnet/corefxlab) DataFrames to UDF APIs ([#277](https://github.com/dotnet/spark/pull/277))
+* Support for ML TF-IDF (Term frequency-inverse document frequency) feature vectorization method ([#394](https://github.com/dotnet/spark/pull/394))
+* Support for TimestampType in `DataFrame.Collect()`, `CreateDataFrame` and UDFs ([#428](https://github.com/dotnet/spark/pull/428))
+* Support for Broadcast Variables ([#414](https://github.com/dotnet/spark/pull/414))
+* Support for ML feature Word2Vec ([#491](https://github.com/dotnet/spark/pull/491))
+* Streamline logging when there is a failure ([#439](https://github.com/dotnet/spark/pull/439))
+
+
+### Breaking Changes
+
+* `SparkSession.Catalog` is changed from a method to a property ([#508](https://github.com/dotnet/spark/pull/508))
+
+### Compatibility
+
+#### Backward compatibility
+
+The following table describes the oldest version of the worker that the current version is compatible with, along with new features that are incompatible with the worker.
+
+<table>
+    <thead>
+        <tr>
+            <th>Oldest compatible .NET for Apache Spark worker</th>
```

Comment:
Oldest compatible Microsoft.Spark.Worker version

Reply:
Made the change